syntax = "proto3";

message IAuthResponse {
    int32 status_code = 1;
    string id = 2;
}
